.travis.yml

```diff
@@ -21,7 +21,6 @@ script:
   - mix credo
```

Comment:
All of these lines can be removed from the travis file because we're doing them when docker builds.
```
- mix format --check-formatted	  
- mix credo
- mix hex.outdated || true
- mix test
```